packages/ui-default/pages/setting.page.tsx

```
@@ -1,29 +1,160 @@
+  const { monaco, registerAction, renderMarkdown } = await load(['yaml']);
+  const KForm = applyVueInReact(form.Form) as any;
+
+  function MarkdownContainer({ source }) {
+    const rendered = React.useMemo(() => {
+      const res = renderMarkdown({ value: source });
+      const value = res.element.innerHTML;
+      res.dispose();
+      return value;
+    }, [source]);
+    // Markdown snippet come from trusted backend code, no need to sanitize here
+    return <div dangerouslySetInnerHTML={{ __html: rendered }} />;
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)